native-netty-plot/pom.xml

```diff
@@ -33,6 +35,67 @@
       <version>3.4.3</version>
     </dependency>
   </dependencies>
+
+    <profiles>
+        <profile>
+            <id>native</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.graalvm.buildtools</groupId>
+                        <artifactId>native-maven-plugin</artifactId>
+                        <version>${native.maven.plugin.version}</version>
+                        <extensions>true</extensions>
+                        <executions>
+                            <execution>
+                                <id>native</id>
+                                <goals>
+                                    <goal>compile-no-fork</goal>
+                                </goals>
+                                <phase>package</phase>
+                            </execution>
+                        </executions>
+                        <configuration>
+                            <imageName>${imageName}</imageName>
+                            <fallback>false</fallback>
+                            <buildArgs>
+                                <buildArg>-H:DashboardDump=fortune -H:+DashboardAll</buildArg>
```

Comment:
Lesia, this is the demo for Native Image isolates, why the name is "fortune" and why do we generate the data for GraalVM dashboard?

Reply:
It was my copy paste mistake. I fixed it!